settings.gradle.kts

```diff
@@ -1,18 +1,24 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+    }
+}
 plugins {
     id("org.gradle.toolchains.foojay-resolver-convention") version "0.5.0"
 }
 rootProject.name = "qpay"
 
+@Suppress("UnstableApiUsage")
```

Comment:
It's more correctly to use `suppress` for repositories. But you can leave as it is

```
dependencyResolutionManagement {
    @Suppress("UnstableApiUsage")
    repositories {
        mavenCentral()
    }
}
```